src/services/metrics-recorder.ts

```diff
@@ -136,29 +134,30 @@ export class MetricsRecorder {
 
       // Update service node quality with cherry picker
       if (serviceNode) {
-        await this.cherryPicker.updateServiceQuality(blockchain, applicationID, serviceNode, elapsedTime, result)
+        await this.cherryPicker.updateServiceQuality(blockchainID, applicationID, serviceNode, elapsedTime, result)
       }
 
       // Bulk insert relay / error metrics
       const postgresTimestamp = new Date()
       const errorValues = [
         postgresTimestamp,
         applicationPublicKey,
-        blockchain,
+        blockchainID,
         serviceNode,
         elapsedTime,
         bytes,
         method,
         error,
       ]
 
+      console.log('Influx Blockchain Tag:', blockchainID)
```

Comment:
Let's remove this since we are logging to a separate bucket and can see it there and so we won't have to do another commit to get rid of it later.